Doc/examples/tree1.nwk

```diff
@@ -0,0 +1 @@
+((A:0.10200,B:0.23000):0.06000,C:0.40000):0.00000;
```

Comment:
This file should not be checked it - if you can see which test is creating the file and leaving it behind, fixing that would be a welcome separate pull request.